src/test/java/teammates/test/cases/webapi/GetCourseActionTest.java

```diff
@@ -178,7 +178,6 @@ protected void testAccessControl_testStudentAccess_shouldPass() {
         verifyInaccessibleWithoutLogin(submissionParams);
         verifyInaccessibleForUnregisteredUsers(submissionParams);
         verifyInaccessibleForInstructors(submissionParams);
-        verifyInaccessibleForAdmin(submissionParams);
```

Comment:
So it should be change to `verifyAccessibleForAdmin`?

Reply:
Yea, should have done this welp thanks